library/src/main/java/com/okta/appauth/android/OktaAppAuth.java

```diff
@@ -565,6 +587,30 @@ private void doAuth(PendingIntent completionIntent, PendingIntent cancelIntent)
                 mAuthIntent.get());
     }
 
+    private void doEndSession(PendingIntent completionIntent, PendingIntent cancelIntent) {
+        Log.d(TAG, "Starting end session flow");
+        try {
+            mAuthIntentLatch.await();
+        } catch (InterruptedException ex) {
+            Log.w(TAG, "Interrupted while waiting for auth intent");
```

Comment:
exception is lost here, we should probably include it in the log message

Reply:
Reworked warmup flow a little bit, so no latch needed anymore